specs/_features/maxeb_increase/capella.py

```diff
@@ -1776,6 +1788,49 @@ def process_execution_layer_withdraw_request(
     ))
 
 
+def is_consolidated(validator: Validator) -> bool:
+   validator.consolidated_to != UNSET_CONSOLIDATED_TO
+
+
+def resolve_slashed_index(state: BeaconState, index: ValidatorIndex) -> ValidatorIndex:
+    if is_consolidated(state.validators[index]):
+        # Recursively resolve consolidated index
+        return resolve_slashed_index(state, state.validators[index].consolidated_to)
+    else:
+        return index
+
+
+def process_consolidation(state: BeaconState, consolidation: Consolidation) -> None:
+    target_validator = state[consolidation.target_index]
+    source_validator = state[consolidation.source_index]
+
+    assert target_validator.exit_epoch == FAR_FUTURE_EPOCH
+    assert source_validator.exit_epoch == FAR_FUTURE_EPOCH
```

Comment:
I think we should add the following checks to the source (exiting) validator (from `process_voluntary_exits`):
```python
# Verify the validator is active
assert is_active_validator(source_validator, get_current_epoch(state))
# Verify the validator has been active long enough
assert get_current_epoch(state) >= source_validator.activation_epoch + config.SHARD_COMMITTEE_PERIOD
```